softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorithation/impl/router/AuthorizationRouterAppConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.softgroup.authorithation.impl.router;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Created by User on 21.03.2017.
+ */
+
+
+
+@Configuration
+@ComponentScan("com.softrgroup.authorization.impl.router")
```

Comment:
I don't understand the question?) I have already handlers for authorization: Login, SmsConfirm, etc.